docs/api_docs/python.rst

```diff
@@ -0,0 +1,214 @@
+
+Suppose we want to see the current state of all parameters in the :ref:`System Control <system_control>` client.
+
+To do so, we can do the following:
+
+.. code-block:: python
+
+    print(module.get_all("system_control"))
+
+Again, you can also treat the returned value as a normal parameter, and store it in a variable. In this case the data is stored as an array.
+
+Set
+^^^^^^^^^^
+Most Python set commands have the format ``module.set("client", "client_entry", value)``. The set function changes the value of the target ``client_entry`` to ``value``. 
+A value set and not saved will not be retained after a power cycle.
+
+Suppose we want to change the :ref:`Propeller Motor Controller's <propeller_motor_controller>` ``timeout`` parameter to 3 seconds.
+
+.. image:: ../_static/api_pics/timeout_entry.png
+
+To do so:
+
+.. code-block:: python
+
+    module.set("propeller_motor_control", "timeout", 3)
+
+Some client entries (such as :ref:`System Control's <system_control>` ``reboot_program``) accept sets without a ``value``. Simply calling ``module.set("client", "client_entry")`` is enough to 
+trigger the desired behavior. So, to reboot your module's program:
+
+.. code-block:: python
+
+    module.set("system_control", "reboot_program")
+
+Set Verify
+^^^^^^^^^^^^
+Additionally to the standard set, you can use the ``set_verify`` function in order to set a new value, and confirm that the value has been set correctly on the module. Set verify 
+calls have the same format as standard set commands, but provide additional optional parameters ``get_values``, ``time_out``, ``retries``, and ``save``. 
+
+* ``get_values``: Specifies values to add in the get message (such as index for some client entries). By default, this is ``None``
+* ``time_out``: A blocking timeout while verifying the set is successful in seconds. By default, this is 0.1s
+* ``retries``: The number of times you would like to retry the set before giving up. By default, this is 5
+* ``save``: Allows you to save the value once the set is confirmed. By default this is ``False``
+
+Save
+^^^^^^^^^^^^
+All Python save commands have the format ``module.save("client", "client_entry")``. The save function takes the currently set entry value, and stores it in the module's persistent memory. 
+Values that are saved are retained on power cycles.
+
+Suppose we want to save the timeout value set above. To do so:
+
+.. code-block:: python
+
+    module.save("propeller_motor_control", "timeout")
+
+Next Steps
+=========================
+As the get, set, and save commands are the basis of all IQUART configuration and control, you now posess all of the base knowledge necessary to begin development 
+with the Vertiq Python API.
+
+A very basic example is provided here. It demonstrates the basics of setting up communication and a module object as well as how to ``set``, ``get``, and ``get_all``.
```

Comment:
Maybe worth providing our standard, "remove props and secure motor disclaimer" here